_config.yml

```diff
@@ -66,6 +66,8 @@ moxieMNotOnCloud: "> **Note**: The MoxieManager plugin is _not_ provided on the
 
 predefinedIconsOnly: "Name of the icon to be displayed. Must correspond to an icon: in the [icon pack](https://www.tiny.cloud/docs/advanced/editor-icon-identifiers/), in a [custom icon pack](https://www.tiny.cloud/docs/advanced/creating-an-icon-pack/), or added using the [`addIcon` API](https://www.tiny.cloud/docs/api/tinymce.editor.ui/tinymce.editor.ui.registry/#addicon)."
 
+liveDemoCSSStyles: "'body { font-family:Helvetica,Arial,sans-serif; font-size:14px }'"
```

Comment:
This will cause issues with inline demos. I'm not sure it's used, but maybe it should be named to highlight it for iframe demos only?

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/8bc36cb9e57a4c890e01fa0a70ebc3484a22a7bb